localization/english/wc_character_interactions_l_english.yml

```
@@ -1,2 +1,7 @@
 ﻿l_english:
- give_magic_eyes_interaction:0 "Give Magic Eyes"
\ No newline at end of file
+ give_magic_eyes_interaction:0 "Give Magic Eyes"
+
+ stasis_trap_already_stunned_first_not:0 "I am not stunned"
+ stasis_trap_already_stunned_first:0 "I am stunned!"
+ stasis_trap_already_stunned_third_not:0 "[CHARACTER.GetShortUIName] is not stunned"
+ stasis_trap_already_stunned_third:0 "[CHARACTER.GetShortUIName] is already stunned"
```

And move it to `wc_character_triggers_l_english.yml`.
```suggestion
 stasis_trap_already_stunned_first_not:0 "I am #bold not#! stunned"
 stasis_trap_already_stunned_first:0 "I am stunned"
 stasis_trap_already_stunned_third_not:0 "[CHARACTER.GetShortUIName] is #bold not#! stunned"
 stasis_trap_already_stunned_third:0 "[CHARACTER.GetShortUIName] is already stunned"
```